core/lib/common/worker.js

```diff
@@ -625,7 +388,7 @@ module.exports = class Worker {
 	 * @category helper
 	 */
 	async rebootDut(target) {
-		this.logger.log(`Rebooting the DUT`);
+		console.log(`Rebooting the DUT`);
```

Comment:
What was the reason for these logging changes?

Reply:
`this.logger`, as far as I could tell, was just a wrapper for `console.log` anyway.